### PR TITLE
[SEC] fetchmail_attach_from_folder: fix unsafe eval

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: fetchmail_attach_from_folder/model/fetchmail_server.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def fields_view_get(self, cr, user, view_id=None, view_type='form',
                 if field.tag == 'field' and field.get('name') in modifiers:
                     field.set('modifiers', simplejson.dumps(
                         dict(
-                            eval(field.attrib['modifiers'],
+                            safe_eval(field.attrib['modifiers'],
```
Comment (sebalix) [inline fetchmail_attach_from_folder/model/fetchmail_server.py:257]: I think it needs to be imported?

**Comment by antespi** [comment]
:+1:

File: fetchmail_attach_from_folder/model/fetchmail_server.py (hunk 1 above)
Comment (gurneyalex) [inline fetchmail_attach_from_folder/model/fetchmail_server.py:257]: @sebalix good catch. It's fixed now.

Comment (sebalix) [inline fetchmail_attach_from_folder/model/fetchmail_server.py:257]: @gurneyalex it is strange that the previous build did not notice it, it was all green https://travis-ci.org/OCA/server-tools/builds/109303252

**Comment by sebalix** [comment]
:+1:

File: fetchmail_attach_from_folder/model/fetchmail_server.py (hunk 1 above)
Comment (gurneyalex) [inline fetchmail_attach_from_folder/model/fetchmail_server.py:257]: @sebalix the module is not ported to v8, so travis does not care :smile_cat:

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**